spec/collectors/statsd.rb

```diff
@@ -0,0 +1,38 @@
+require "rspec_profiling/config"
```

Comment:
Please use standard .rb spacing - 2 spaces per indent (not tabs!)

Reply:
ugh i hate tabs, I'll look into this